apps/backend-ops/src/select-environment.ts

```diff
@@ -18,7 +18,10 @@ function SAKIsCorrect(projectId: string) {
   try {
     key = getServiceAccountObj(process.env[SAKKeyName]);
     console.log('Current SAK projectId:', key.project_id);
-  } catch (e) { void 0; }
+  } catch (e) {
+    console.log('Was unable to detect previously set SAK')
```

Comment:
```suggestion
    console.log('Was unable to detect previously set SAK');
```

You know I always check for `;` please add them before I ask